app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CategoriesController < ApplicationController
+  decorates_assigned :category, :books
+
+  def index
+    @pagy, @books = pagy(BookSorter.call(sort_params))
+  end
+
+  def show
+    @category = Category.find(params[:id])
+    @pagy, @books = pagy(BookSorter.call(sort_params, @category.books))
+  end
+
+  private
+
+  def sort_params
+    params.permit(:status, :category, :commit, :id)
```

Comment:
why do we need permit category and commit from params?

Reply:
![Screenshot from 2019-12-16 10-49-25](https://user-images.githubusercontent.com/47256098/70892402-c9b6c800-1ff1-11ea-8dd4-bdee4a9c43a4.png)
![Screenshot from 2019-12-16 10-50-12](https://user-images.githubusercontent.com/47256098/70892472-e7842d00-1ff1-11ea-9682-f4129d318381.png)
![Screenshot from 2019-12-16 10-51-02](https://user-images.githubusercontent.com/47256098/70892523-02ef3800-1ff2-11ea-8533-3a9c4cac9146.png)